.travis.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   include:
     - stage: 'security audit and linting'
       language: node_js
-      node_js: 'stable'
```

Comment:
I don't understand why this didn't work. This is the error that Travis gave:
```
Now using node v19.3.0 (npm v)
node: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.27' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.25' not found (required by node)
node: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by node)
The command "npm config set spin false" failed and exited with 1 during .
```

But I guess having it set to 16.14.2 is OK too since that's what I run locally on my machine.